pymatgen/core/xcfunc.py

```diff
@@ -101,13 +101,13 @@ class type_name(NamedTuple):
         (xcf.GGA_X_B88, xcf.GGA_C_LYP): type_name("GGA", "BLYP"),
     }
 
-    del type_name
+    # del type_name
```

Comment:
Delete `type_name` right after usage seem to break `TYPE_CHECKING` for `ClassVar`, need to confirm if this would incur any significant memory penalty.

Reply:
i wouldn't worry about mem overhead here. the `type_name` variable goes out of scope and will be garbage-collected on line 121 anyway

Reply:
Thanks!